packages/render-fidelity-tools/src/artifact-creator.ts

```diff
@@ -83,6 +84,15 @@ export class ArtifactCreator {
       const analysisResults =
           await this.analyze(screenshot, goldens, scenario, dimensions);
 
+      // model-viewer is in 0 index
```

Comment:
In interviews people keep telling me not to use these kind of "magic index" because it's not very readable. Do you feel like this is a problem? Do you have better ideas to make this more readable?

Reply:
Yes, the better thing to do is `const modelViewerIndex = 0`, then use that below. If you can remove a comment by adding a name, that's almost always clearer.